### PR TITLE
feat(centralized arbitrator dashboard)

### PR DESCRIPTION
**Pull request by 0xferit**
- Added event handler for centralized arbitrator dashboard which send an email on new disputes if user wants an email.

- Added `@kleros/kleros-interaction` dependency. I needed it to instantiate a CentralizedArbitrator and call `.owner()`

### CODE REVIEW

File: src/event-handlers/centralized-arbitrator-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+const centralizedArbitrator = require('@kleros/kleros-interaction/build/contracts/CentralizedArbitrator.json')
+
+const _web3 = require('../utils/web3')
+const _sendgrid = require('../utils/sendgrid')
+const dynamoDB = require('../utils/dynamo-db')
+
+module.exports.post = async (
+  centralizedArbitratorAddress,
```
Comment (satello) [inline src/event-handlers/centralized-arbitrator-dashboard.js]: centralizedArbitratorAddress will be passed in the event body.

So 
```
const centralizedArbitratorAddress = event.body.contractAddress
```

File: src/event-handlers/centralized-arbitrator-dashboard.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        error: 'No event log passed in body.'
+      })
+    })
+  }
+
+  const centralizedArbitratorInstance = new _web3.eth.Contract(
+    centralizedArbitrator.abi,
+    centralizedArbitratorAddress,
+    {
+      gasPrice: 20000000000
```
Comment (satello) [inline src/event-handlers/centralized-arbitrator-dashboard.js]: is this necessary?

File: src/event-handlers/centralized-arbitrator-dashboard.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    to: emailAddress,
+    from: {
+      name: 'Kleros',
+      email: 'contact@kleros.io'
+    },
+    templateId: 'd-8e6dd684d23447a8a5051adf69396d58',
+    dynamic_template_data: {
+      name: fullName,
+      arbitratorAddress: centralizedArbitratorAddress,
+      disputeID: body._disputeID,
```
Comment (satello) [inline src/event-handlers/centralized-arbitrator-dashboard.js]: you wont have disputeID in the body. The body is the event log

File: src/event-handlers/centralized-arbitrator-dashboard.js (hunk 2 above)
Comment (0xferit) [reply src/event-handlers/centralized-arbitrator-dashboard.js]: Probably not.

File: src/event-handlers/centralized-arbitrator-dashboard.js (hunk 3 above)
Comment (0xferit) [reply src/event-handlers/centralized-arbitrator-dashboard.js]: https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#contract-events-return

This is what I'll get right?

so I can obtain disputeID with: `body.returnValues._disputeID`

Comment (satello) [reply src/event-handlers/centralized-arbitrator-dashboard.js]: Ah yep if its an arg you can get it from returnValues

**Pull request closed without merging**